hal/aarch64/zynqmp/zynqmp.c

```diff
@@ -898,6 +948,15 @@ unsigned int hal_getBootReason(void)
 	return zynq_common.resetFlags;
 }
 
+void _zynqmp_config_axi_width(void)
+{
+	/* Both AXI interfaces in Full Power Domain configured to 32-bit width */
+	*(zynq_common.fpd_slcr + fpd_slcr_afi_fs) = (FPD_AXI_S0_WIDTH << 8)
+		| (FPD_AXI_S1_WIDTH << 10);
```

Comment:
Nit: Nicer to end the line with the operator
```suggestion
	*(zynq_common.fpd_slcr + fpd_slcr_afi_fs) = (FPD_AXI_S0_WIDTH << 8) |
		(FPD_AXI_S1_WIDTH << 10);
```